data/map.txt

```diff
@@ -21855,8 +21855,8 @@ system "Tania Australis"
 	fleet "Small Northern Merchants" 1000
 	fleet "Large Northern Merchants" 2000
 	fleet "Small Republic" 3000
-	fleet "Small Northern Pirates" 4000
-	fleet "Large Northern Pirates" 9000
+	fleet "Small Southern Pirates" 4000
+	fleet "Large Southern Pirates" 9000
```

Comment:
Why? These systems have northern merchants, why would they then have southern pirates?

Reply:
Because they are in the Dirt Belt, connected to systems with Southern Pirates, but not to any systems with Northern Pirates. So N pirates would have nowhere to come from or go to, S pirates would. See also first post.

Reply:
Porrima and Regulus are the only two systems where southern pirates currently overlap with northern merchants, one of those being thanks to #4038, and this PR would remove them from overlapping in Regulus. Due to this, I'd say that the addition of northern merchants in Porrima could be reverted, and these pirate fleets should be swapped back to northern. (Notably, northern pirates don't overlap with southern merchants anywhere, and core pirates only overlap with northern merchants in Cardax.)

I don't see disconnectedness as an issue here seeing as how northern pirates are generally disconnected in this region anyways. Core and northern pirates also appear around southern pirate systems, so clearly pirates are doing something to evade detection and make their way to these systems, making them *appear* disconnected.